src/inverseRobot.js

```diff
@@ -4,10 +4,26 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+
+  for (const robotKey in robot) {
+    const key = robot[robotKey];
+    const value = robotKey;
+
+    obj[key] = value;
+  }
+
+  const robotLength = Object.keys(robot).length;
+  const objLength = Object.keys(obj).length;
+
+  if (robotLength === objLength) {
+    return obj;
+  }
+
+  return null;
```

Comment:
better to check it on each iteration to return null on the first repeated value and reduce the algorithm complexity